<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #c7e3f1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126,197,227);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>731</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 33, 255, 1);
background-color: none;</string>
    </property>
    <property name="text">
     <string>STUDENT SYSTEM MANAGEMENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
border-image: url(:/newPrefix/school.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126,197,227);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#delete_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#delete_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#delete_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#refresh_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#refresh_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#refresh_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#update_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#update_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#update_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rollcall_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#rollcall_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#rollcall_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#rollcall_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Create a code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#add_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#add_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#add_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="show_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#show_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#show_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#show_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="absence_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#absence_btn{
	background-color: rgb(255,255,255);
	border-radius: 15px;
	color: black;
}
QPushButton#absence_btn:hover{
	background-color: rgb(255,255,255,210);
	border-radius: 15px;
	color: black;
}
QPushButton#absence_btn:pressed{
	background-color: rgb(255,255,255,210);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Absence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 33, 255, 1);
color: white;</string>
    </property>
    <property name="text">
     <string>          MENU</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>821</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 33, 255, 1);
color: white;
</string>
    </property>
    <property name="text">
     <string>                                                                                       DISPLAY</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="show_table">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>90</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="library_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>710</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#library_btn{
	background-color: rgba(7, 255, 120, 1); 
	border-radius: 25px;
	color: black;
}
QPushButton#library_btn:hover{
	background-color: rgba(7, 255, 17, 1);
	border-radius: 25px;
	color: black;
}
QPushButton#library_btn:pressed{
	background-color: rgba(7, 255, 17, 1);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Library Management</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>710</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#logout_btn{
	background-color: rgba(7, 255, 120, 1); 
	border-radius: 25px;
	color: black;
}
QPushButton#logout_btn:hover{
	background-color: rgba(7, 255, 17, 1);
	border-radius: 25px;
	color: black;
}
QPushButton#logout_btn:pressed{
	background-color: rgba(7, 255, 17, 1);
	padding-top: 5px;
	padding-left: 5px;
}</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res_school.qrc"/>
 </resources>
 <connections/>
</ui>
